docs/docs/1.7.x/policies/proxy-template.md

```diff
@@ -513,7 +513,7 @@ spec:
 #### HTTP Filter
 
 Modifications that are applied on [HTTP Filters](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/filter/http/http) that are part of [Listeners](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto#config-listener-v3-listener) resource.
-Modifications are applied on all [HTTP Connection Managers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html#http-connection-manager-proto) in the Listener.
+Modifications applies on all [HTTP Connection Managers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html#http-connection-manager-proto) in the Listener.
```

Comment:
This does not sound right I think. Shouldn't this be:

```suggestion
Proxy Template applies modifications on all [HTTP Connection Managers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html#http-connection-manager-proto) in the Listener.
```

It's a bit tricky because the subject is implicit... I don't know, maybe @parkanzky or @michaelbeaumont would know.

Reply:
Wait, why are these changes in this PR at all? :thinking: 

```suggestion
Modifications that Kuma applies on all [HTTP Connection Managers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html#http-connection-manager-proto) in the Listener.
```

Since these are sort of dependent clauses that explain what each modification type is. Here, what `httpFilter` modifications are.

Reply:
I've changed the URL because it was invalid, and I vale starts to complain, so I had to change it

Reply:
That was fixed in master I believe, there's no change to the URL here.